WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/IconListItem.swift

```diff
@@ -0,0 +1,67 @@
+import Kingfisher
+import SwiftUI
+
+/// Represents an icon to be displayed. Base64 or Remote (from web)
+enum Icon {
+    /// Base64 icon
+    case base64(UIImage)
+
+    /// Icon that comes from an URL
+    case remote(URL)
+
+    /// Return an Image for base64 or a FKImage in case of a remote one
+    @ViewBuilder
+    func getImage() -> some View {
+        switch self {
+        case .base64(let image):
+            Image(uiImage: image)
+                .resizable()
+        case .remote(let url):
+            KFImage(url)
+                .resizable()
+        }
+    }
+}
```

Comment:
This looks nice!
I wonder if this should live under the `IconListItem` namespace? 🤔 

Anyway, I think it's fine like this for now.

Reply:
I think it makes sense! I'll change it on this PR if you don't mind :)
Done! ✅